i18n/index.js

```diff
@@ -1,8 +1,12 @@
 var en = require('./translations.en.json');
+var es = require('./translations.es.json');
+var de = require('./translations.de.json');
 
 const i18n = {
   translations: {
     en,
+    de,
+    es,
   },
   defaultLang: 'en',
   useBrowserDefault: false,
```

Comment:
should we turn this to "true" to assign the most appropriate language automatically?